server/test/services/export/AbstractExporterTest.java

```diff
@@ -76,11 +77,13 @@ public abstract class AbstractExporterTest extends ResetPostgres {
   protected ApplicationModel applicationFive;
   protected ApplicationModel applicationSix;
   protected ApplicationModel applicationSeven;
+  private ApplicationStatusesRepository apprepo;
 
   @Before
   public void setup() {
     programAdminApplicationService = instanceOf(ProgramAdminApplicationService.class);
     programService = instanceOf(ProgramService.class);
+    apprepo = instanceOf(ApplicationStatusesRepository.class);
```

Comment:
**suggestion:** Can we camel case this? Should be `appRepo` instead of `apprepo`

Reply:
renamed it :)